lib/Yam/ControlCenterPage.pm

```diff
@@ -0,0 +1,368 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Control Center
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::ControlCenterPage;
+use parent 'Yam::PageBase';
+use strict;
+use warnings;
+use testapi;    # needed because of bsc1206966
+
+sub init {
+    my $self = shift;
+    $self->{lbl_yast_cc} = $self->{app}->label({label => 'YaST Control Center'});
+    $self->{slb_groups} = $self->{app}->selectionbox({id => 'groups'});
+    $self->{slb_progs} = $self->{app}->selectionbox({id => 'progs'});
+    $self->{btn_help} = $self->{app}->button({id => 'help'});
+    $self->{btn_run} = $self->{app}->button({id => 'run'});
+    $self->{btn_quit} = $self->{app}->button({id => 'quit'});
+    return $self;
+}
+
+sub get_control_center_page {
+    my ($self) = @_;
+    die 'YaST Control Center is not displayed' unless $self->{lbl_yast_cc}->exist();
+    return $self;
+}
+
+sub apply_workaround_bsc1206966 {
```

Comment:
you could add `# workarounds` to be consistent with comments above this method name.